src/state/state.go

```diff
@@ -40,21 +43,21 @@ type ObjectState interface {
 
 type BranchState struct {
 	*manifest.BranchManifest
-	Remote *model.Branch `validate:"dive"`
-	Local  *model.Branch `validate:"dive"`
+	Remote *model.Branch `validate:"omitempty,dive"`
```

Comment:
Nevalidovalo sa to, tak som az teraz prisiel na to, ze je tu chybny tag, ... hodnota sa ma validovat, iba ked je zadana ... objekt nemusi existovat v Remote aj Local stave.